corehq/apps/performance_sms/message_sender.py

```diff
@@ -0,0 +1,22 @@
+from collections import namedtuple
+from corehq.apps.performance_sms.parser import get_parsed_params
+from corehq.apps.performance_sms.query_engine import QueryEngine, QueryContext
+from corehq.apps.sms.api import send_sms
+
+
+MessageResult = namedtuple('MessageResult', ['user', 'message'])
+
+
+def send_messages_for_config(config, actually_send=True):
+    query_engine = QueryEngine(template_vars=config.template_variables)
+    params = get_parsed_params(config.template)
+    sent_messages = []
+    for user in config.group.get_users():
+        if user.phone_number:
+            query_context = QueryContext(user, config.group, template_vars=config.template_variables)
+            message_context = query_engine.get_context(params, query_context)
+            message = config.template.format(**message_context)
+            if actually_send:
+                send_sms(config.domain, user, user.phone_number, message)
```

Comment:
Thanks for pulling out those functions into utils, that looks good.  There's just a couple things that need to be updated here:

- we should check for `isinstance(phone_number, VerifiedNumber)`, and use `send_sms_to_verified_number` instead of `send_sms` if it's True
- if it's False, the call to `send_sms` should use `phone_number` (not `user.phone_number`)